contracts/VaultV3.vy

```diff
@@ -15,12 +15,13 @@ interface IStrategy:
     def totalAssets() -> (uint256): view
     def convertToAssets(shares: uint256) -> (uint256): view
     def convertToShares(assets: uint256) -> (uint256): view
-    def migrate(strategy: address): nonpayable
-    def tend(): nonpayable
 
 interface IAccountant:
     def report(strategy: address, gain: uint256, loss: uint256) -> (uint256, uint256): nonpayable
 
+interface IQueueManager:
+    def withdraw_queue() -> (DynArray[address, 10]): nonpayable
```

Comment:
with a seperate QueueManager contract we would probably want to use the same one for the vault, a potential router, or any direct front end integration etc. So would want it to take the address of vault as a parameter so anyone can view the queue

Reply:
https://github.com/flashfish0x/yearn-vaults-v3/pull/1/commits/13f0152deac86c0041ddce48f97557f0709f58e6